api/pkg/cli/cmd/root.go

```diff
@@ -68,7 +68,7 @@ func Root(cli app.CLI) *cobra.Command {
 	)
 
 	cmd.PersistentFlags().StringVar(&apiURL, "api-server", "", "Hub API Server URL (default 'https://api.hub.tekton.dev' for 'tekton' type; default 'https://artifacthub.io' for 'artifact' type).\nURL can also be defined in a file '$HOME/.tekton/hub-config' with a variable 'TEKTON_HUB_API_SERVER'/'ARTIFACT_HUB_API_SERVER'.")
-	cmd.PersistentFlags().StringVar(&hubType, "type", "tekton", "The type of Hub from where to pull the resource. Either 'artifact' or 'tekton' (default 'tekton')")
+	cmd.PersistentFlags().StringVar(&hubType, "type", "tekton", "The type of Hub from where to pull the resource. Either 'artifact' or 'tekton'")
```

Comment:
not related to this patch but since I saw this, hence commenting it here:
I guess we should rename `artifact` to `artifacthub` as word `artifact` is very very generic. Can you please update this?
```suggestion
	cmd.PersistentFlags().StringVar(&hubType, "type", "tekton", "The type of Hub from where to pull the resource. Either 'artifacthub' or 'tekton'")
```

Reply:
I agree, but for Tekton we are not adding `TektonHub` so I think it should be fine
wdyt ?

Reply:
acha haan it's fine